exercises/06-load-balancers.md

```diff
@@ -0,0 +1,58 @@
+# `06` Load Balancers
+
+Modern deployments in general, and Kubernetes deployments in particluar, often employ load balancers to distribute load, improve uptime, and to increase security.
+
+Our load balancers are highly available, as they run in pairs on separate hosts. In addition to being manageable by API, Terraform, and Ansible, they also come with Cloud Controller Manager (CCM) integration. This means they can easily be integrated into Kubernetes.
+
+## 💡 Background Information
+
+### References
+
+* [API Documentation](https://www.cloudscale.ch/en/api/v1#load-balancers)
+* [Ansible cloud collection](https://docs.ansible.com/ansible/latest/collections/cloudscale_ch/cloud/).
+* [Terraform provider](https://registry.terraform.io/providers/cloudscale-ch/cloudscale/latest/docs).
+* [Cloudscale Cloud Controller Manager](https://github.com/cloudscale-ch/cloudscale-cloud-controller-manager)
+* [K8test](https://github.com/cloudscale-ch/k8test)
+* [Blog Post: K8s Cloud Controller Manager for cloudscale](https://www.cloudscale.ch/en/news/2024/03/05/cloud-controller-manager)
+* [Blog Post: Load Balancer "as a Service"](https://www.cloudscale.ch/en/news/2023/04/28/load-balancer-as-a-service)
+
+## 🎓 Exercises
+
+### `06.01` - Load Balancer Components
+
+Load balancers include components, read the documentation to understand what these end-points manage:
```

Comment:
Please change to "Load balancers consist of various components. Read..."
If "load balancer" does not have a dash, then "end points" does not have one either.